eo-maven-plugin/src/test/java/org/eolang/maven/BinarizeParseMojoTest.java

```diff
@@ -137,11 +143,14 @@ void createsCorrectRustProject(@TempDir final Path temp) throws Exception {
         final Map<String, Path> res = maven
             .execute(new FakeMaven.BinarizeParse())
             .result();
+        final String prefix = temp.resolve("target").toString()
+            .toLowerCase(Locale.ENGLISH)
+            .replaceAll("[^a-z0-9]", "x");
         final String cargo = "target/Lib/Cargo.toml";
         final String lib = "target/Lib/src/lib.rs";
         final String module = String.format(
-            "target/Lib/src/%s.rs",
-            "fooorgoeolangocustomocreatesoobjector03a6002e006f00720067002e0065006f006c0061006e0067002e0063007500730074006f006d002e0063007200650061007400650073002d006f0062006a006500630074002e0072"
+            "target/Lib/src/%s1.rs",
```

Comment:
@levBagryansky You can use just `%s` here.

Reply:
@volodya-lombrozo Why is it better? Do you suggest replace `target/Lib/src` by `%s`?